### ISSUE TITLE
Most recent update issues

### ISSUE THREAD

**Issue by kju** [deanoemcke/thegreatsuspender#990]
Please complete the following information when submitting a feature request or bug report.
 * Extension version: v7.1.0
 * Browser name & version: Chrome Version 76.0.3809.100 (Official Build) (64-Bit)
 * Operating system & version: Windows 10 Home 64b - 10.0.18362 Build 18362

1. In auto update dialog it offered to save open sessions. In the session.txt only 3 entries are present of ~3000 tabs. Note: these 3 were some old tabs - not the oldest, nor the first or last tabs.
2. During the auto update the updater seems to have crashed. It remained with 600 tabs open.
3. Chrome reported the extension to be corrupted and offered repair option.
4. Restarted chrome and it still had the 600 tabs.
5. After the repair option applied, the extension appears to work fine again and is up-to-date,
6. The extension had not recent sessions in session management (before I imported the bugged export, and saved the remaining 600).

As it seems to have lost mostly old sessions, and as I still have a backup 8 days ago from Session Buddy, I doesnt affect me much, yet the whole update process need a lot more attention and testing it seems.

As Session Buddy can store also suspended tabs (you cannot open directly but you still have the url), you may look into it, to make sure you offer users save backups for your extension update.

Thanks for your work regardless of this big mishap here!

PS: If you want to recover your tabs, try to export the history:
https://chrome.google.com/webstore/detail/history-export/lpmoaclacdaofhlijejogfldmgkdlglj
https://www.makeuseof.com/tag/export-chrome-browsing-history/

**Comment by jonibahu** [comment]
Hi

Did u manage to import the bugged session into the Session Manager by coying the old files into the folder, or did you press the "import" button and imported it that way???


I was trying to get my old "Session manager data" for the new version.
But it seem the new version does not read the "old" session manager data for some reason.

**Comment by liamjohnston** [comment]
The importer has a bug unfortunately :(
HOWEVER you should be able to successfully import a Great Suspender session directly into session buddy. I have tested this and it worked for me :)

**Comment by usmonster** [comment]
> 2. During the auto update the updater seems to have crashed. It remained with 600 tabs open.

This is the issue discussed on #988, #987, #526, and similar issues. It's actually [a bug in the current version of Chrome](https://crbug.com/977805) v76 and should be fixed in a coming update.

> 5. After the repair option applied, the extension appears to work fine again and is up-to-date,
> 6. The extension had not recent sessions in session management (before I imported the bugged export, and saved the remaining 600).

This is "normal" since "Repair" actually wipes all the extension settings, including saved sessions.

**Comment by kju** [comment]
Thanks. 

If its known and frequent issue, the extension should have big warning signs before updating and encourage/force the user to export all relevant data of the extensions (settings, saved sessions, open tabs).

**Comment by Anyasia** [comment]
Just popping in to note that I had the same problem with the exporter not actually exporting my tabs. When I used the backup option in the update dialog, it included only about 20 tabs in the `session.txt`. I then attempted to manually export the session using the main Great Suspender window, and it included only a single tab in `session.txt`—the one currently-unsuspended tab.

Noticing this, I made a manual backup of my 176 open tabs before updating, but luckily it didn't seem to actually lose any tabs for me during the update.
